### ISSUE TITLE
ENJOYING_THIS_PROJECT="SayThankYou"

### ISSUE THREAD

**Issue by pahakalle** [sergiotapia/magnetissimo#111]
Here is the place to say thank you to folks who are working hard on this project (unless other place to do so exists).

« One of the greatest ways to show your appreciation to open source projects you enjoy is to open an issue that let people say thank you »

Cheers!

**Comment by Bechrissed** [comment]
Well, let me be the first: Thank You!
 
A lot of work needs to be done but the idea and work done so far is awesome! 

PS. why "no javascript"? I love JS! :)

**Comment by skwerlman** [comment]
> A lot of work needs to be done but the idea and work done so far is awesome!

Please feel free to give us ideas for what to add/do next!

> PS. why "no javascript"? I love JS! :)

As I understand it, we want to try to keep the codebase simple; introducing JS means introducing a lot of complexity, since its a totally different language with its own dependencies, build tools, etc.

**Comment by Bechrissed** [comment]
Hi @skwerlman,

I think the first thing to be done should be updating the docs. It wasn't so easy to get Magnetissimo up and running on Ubuntu 16.04 server.

Next thing for me would be to add more features to the web interface (eg: categories, tags, filters etc).

I must say i've never heard of erlang/elixir before I started using this project. I will read more the coming days/weeks and hopefully I can chip in and contribute some code.

PS. I also wanted to have rarbg.to added as a source but I see there's already a PR open for that.

**Comment by skwerlman** [comment]
> I think the first thing to be done should be updating the docs. It wasn't so easy to get Magnetissimo up and running on Ubuntu 16.04 server.

Yeah, the wiki needs a bit of restructuring, and also probably some articles on specific topics (how to enable/disable specific crawlers etc)

> I must say i've never heard of erlang/elixir before I started using this project. I will read more the coming days/weeks and hopefully I can chip in and contribute some code.

I actually started contributing to this because I wanted to learn elixir haha. I'd say the easiest way to get started would be to add a new site crawler by copying/modifying an existing one.

> PS. I also wanted to have rarbg.to added as a source but I see there's already a PR open for that.

That PR is written to work with a much older version; a good first PR might be to modernize it?

> Next thing for me would be to add more features to the web interface (eg: categories, tags, filters etc).

This is probably deserving of its own issue, though it is listed on the Future Features wiki page.

**Comment by tchoutri** [comment]
Yes, most of what is written in the Wiki must be handled with care.  
@Bechrissed a great way to start contributing to the project would be to describe how **you** managed to set it up on Ubuntu 16 :)

**Comment by sergiotapia** [comment]
Thanks @pahakalle for the kind words. I hope you enjoyed using it. 

I'm finally in a good spot time-wise where my work is in Elixir and my personal project Magnetissimo is in Elixir. Context switching will be in my favor so I'll be able to work on this more and bring it up to latest and greatest Elixir/Phoenix developments. Big thanks to everybody who helped out with the codebase. 

Now I get to work!

**Comment by sergiotapia** [comment]
@pahakalle We're back in business! Master is now fine, cleaner and with a shiny new coat of paint courtesy of Bulma.

**Comment by skwerlman** [comment]
hey i might be able to contribute to this occasionally starting in a week or so, though i do have a few other personal projects to work on. v grateful to this project for introducing me to elixir!
